### PR TITLE
Added a daily task that publishes to all the registered veggies twitter ...

### PR DESCRIPTION
**Pull request by itayadler**
...accounts a tweet regarding price updates

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -6,4 +6,15 @@ def self.admin_only(options = {})
...
+  def callback
+    access_token = request.env['omniauth.auth'][:credentials][:token]
+    access_secret = request.env['omniauth.auth'][:credentials][:secret]
+    options = {
+      access_token: access_token,
+      access_secret: access_secret
+    }
+
+    #rendering the credentials to the client so an admin could later on add the credentials to the twitter user model
+    render text: options
```
Comment (ShayDavidson) [inline app/controllers/application_controller.rb]: sorry for being petty but we're mixing hash rockets and `:` notation :P

File: app/models/twitter_account.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class TwitterAccount < ActiveRecord::Base
+
+  belongs_to :veggie
+
+  def veggies_by_general_name
+    #In here i'm assuming that the first word of the raw veggie name is the general veggie.
```
Comment (ShayDavidson) [inline app/models/twitter_account.rb:6]: `Assuming that the first word of...`

File: app/services/veggies_publish/twitter_publisher.rb [hunk 3]
```diff
@@ -0,0 +1,32 @@
+require 'twitter_poster'
+require 'active_support/number_helper'
+
+module VeggiesPublish
+  TWEET_PREFIX = 'רק היום'
+  TWEET_VEGGIE_TEMPLATE = "%{name} ב%{price}"
```
Comment (ShayDavidson) [inline app/services/veggies_publish/twitter_publisher.rb]: add a dash between the `ב` and the price

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (ShayDavidson) [inline app/controllers/application_controller.rb]: doesn't this make our secret public? also, lets move this to another controller (`auth`?)

**Comment by ShayDavidson** [comment]
Awesome!

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (itayadler) [inline app/controllers/application_controller.rb]: No because it only prints the user creds and only if he has authenticated

Comment (ShayDavidson) [inline app/controllers/application_controller.rb]: misinterpreted  the `request.env` simply as `env`.

Comment (itayadler) [inline app/controllers/application_controller.rb]: When will u see the light :)

**Pull request merged**